index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en-GB" >
+<head>
+	
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Damian HOW</title>
+	<link rel="stylesheet" href="styles.css" type="text/css" />
+	<link rel="stylesheet" href="normalize.css" type="text/css" />
+	<link href="https://fonts.googleapis.com/css2?family=Azeret+Mono&display=swap" rel="stylesheet" />
+	
+</head>
+
+<body>
+	<div id="page-container" >
+		<div id="content-wrap" >
+			<header>
+				<div class="logo" >
+					<img src="images/logofinal.jpg" alt="Logo" />
+				</div>
+				<div class="title" >
+					<h3>Damian's House of Wonders</h3>
+				</div>
+				<div class="hidden" >
+					<img src="images/rsz_logo.jpg" alt="Hidden logo" />
+				</div>
+			</header>
+			<nav>
+				<div class="nav-links" >
+					<a href="index.html" >home</a>
+					<a href="about.html" >about</a>
+					<a href="portfolio.html" >portfolio</a>
+					<a href="contact.html" >contact</a>
+				</div>
+			</nav>	
+			<main>
+				<img src="/Users/ioacademy/Documents/reductito.github.io/images/rsz_20171104_165733.jpg" alt="Crying Damian" />
+				<p>Hi there, my name is Damian Bell. I am a dashingly handsome and shockingly articulate aspiring developer, currently studying at IO Academy. In my house of wonders you will be able to spy on all of the awesome projects I work on and learn a bit more about me.</p>
+				<img src="/Users/ioacademy/Documents/reductito.github.io/images/rsz_20171104_165947.jpg" alt="Happy Damian" />
```

Comment:
img url

Reply:
I am going through and changing them but just to explain why the full link is there, it's because on mac i usually use the right click and then hold options to copy file as pathname, so the longer URLs were actually a result of me being lazy not me typing it all out